maven-failsafe-plugin/src/test/java/org/apache/maven/plugin/failsafe/RunResultTest.java

```diff
@@ -64,6 +69,49 @@ public void testSkipped() throws Exception {
         writeReadCheck(new RunResult(3, 2, 1, 0, null, true));
     }
 
+    @Test
+    public void testFlakes() throws Exception {
+        writeReadCheck(new RunResult(3, 2, 1, 0, 2, null, true));
+    }
+
+    @Test
+    public void testLegacyDeserialization() throws Exception {
```

Comment:
can you be more specific about the exception thrown?

Reply:
Hey, yeah I could but it's a test method, I'm not convinced it's necessary to be specific here. I was following the existing code style in this particular Test class. I think in this case it may be better to be consistent with the original style.

Reply:
I 100% disagree. Consistency with imperfect code is **not** a virtue. New code should follow best practices, even when the old code does not.